### PR TITLE
Docker php fpm: Clear cache after installation

### PR DESCRIPTION
**Pull request by quartje**
Delete the symfony cache after the installation

### CODE REVIEW

File: docker/php-fpm/Dockerfile [hunk 1]
```diff
@@ -26,6 +26,7 @@ RUN apk add --no-cache libxml2-dev \
     SYMFONY_ENV=prod composer run-script symfony-scripts && \
     chown -R www-data /var/www/html/var/cache && \
     chown -R www-data /var/www/html/var/logs && \
-    chown -R www-data /var/www/html/var/sessions
+    chown -R www-data /var/www/html/var/sessions && \
+    rm -rf /var/www/html/var/cache/prod/
```
Comment (MKodde) [inline docker/php-fpm/Dockerfile:30]: Is the Symfony cache warmed back up at a later stage? Otherwise this makes not much sense to me.

Comment (quartje) [reply docker/php-fpm/Dockerfile:30]: The configuration is applied when the docker image starts up. The first http hit will then create the correct cache right?

Comment (MKodde) [reply docker/php-fpm/Dockerfile:30]: I'm not fully sure that also works in `prod` mode, but apparently it does.

**Review by MKodde — APPROVED** [review]

File: docker/php-fpm/Dockerfile (hunk 1 above)
Comment (quartje) [reply docker/php-fpm/Dockerfile:30]: It does!

**Pull request merged**